### PR TITLE
Synapse 1.52

### PR DESCRIPTION
**Pull request by babolivier**
Preview: https://pr1259--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/content/blog/2022/01/2022-02-08-synapse-1.52-released.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+While preparing the release of Synapse 1.52, we have investigated the impact of this vulnerability on Synapse. We came to the conclusion that it does not affect Synapse. We however advise server administrators to ensure they use an up-to-date version of the library as a matter of good practice.
+
+For instances installed with `pip`, the library can be updated with `pip install --upgrade Twisted`. For instances installed with the `matrixdotorg/synapse` Docker image or the Debian packages from `packages.matrix.org`, just updating to Synapse 1.52.0 should be enough as these images and packages are already used an updated library.
+
+It is also worth noting that a release candidate for [Twisted 22.2](https://github.com/twisted/twisted/releases/tag/twisted-22.2.0rc1) has been published, with a fix for a potential denial of service vulnerability with SSH. Administrators of Synapse homeservers that have the [manhole](https://matrix-org.github.io/synapse/v1.52/manhole.html) feature enabled (which is the only feature of Synapse using SSH) are encouraged to ensure access to the manhole is correctly restricted (e.g. by preventing non-local shell access).
```
Comment (anoadragon453) [inline gatsby/content/blog/2022/01/2022-02-08-synapse-1.52-released.mdx:19]:
```suggestion
It is also worth noting that a release candidate for [Twisted 22.2](https://github.com/twisted/twisted/releases/tag/twisted-22.2.0rc1) has been published, with a fix for a potential denial of service vulnerability with SSH. Administrators of Synapse homeservers that have the [manhole](https://matrix-org.github.io/synapse/v1.52/manhole.html) feature enabled (which is the only feature of Synapse using SSH) are encouraged to ensure access to the manhole is correctly restricted (e.g. by preventing access from external locations).
```

"non-local shell access" was a confusing phrase for me.

File: gatsby/content/blog/2022/01/2022-02-08-synapse-1.52-released.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+[Synapse 1.52](https://github.com/matrix-org/synapse/releases/tag/v1.52.0) is out! Here's what's new with this week's release.
+
+## Twisted security release
+
+The team behind [Twisted](https://twistedmatrix.com/), which is the main framework Synapse uses under the hood, recently released [Twisted 22.1](https://github.com/twisted/twisted/releases/tag/twisted-22.1.0). This version fixes a [security vulnerability](https://github.com/twisted/twisted/security/advisories/GHSA-92x2-jw7w-xvvx) within the Twisted library.
+
+While preparing the release of Synapse 1.52, we have investigated the impact of this vulnerability on Synapse. We came to the conclusion that it does not affect Synapse. We however advise server administrators to ensure they use an up-to-date version of the library as a matter of good practice.
+
+For instances installed with `pip`, the library can be updated with `pip install --upgrade Twisted`. For instances installed with the `matrixdotorg/synapse` Docker image or the Debian packages from `packages.matrix.org`, just updating to Synapse 1.52.0 should be enough as these images and packages are already used an updated library.
```
Comment (callahad) [inline gatsby/content/blog/2022/01/2022-02-08-synapse-1.52-released.mdx:17]:
```suggestion
For instances installed with `pip`, the library can be updated with `pip install --upgrade Twisted`. For instances installed with the `matrixdotorg/synapse` Docker image or Debian packages from `packages.matrix.org`, updating to Synapse 1.52.0 is sufficient, as these images and packages include up-to-date versions of all dependencies.
```

**Review by anoadragon453 — APPROVED** [review]
Looks good overall! Just a couple wording things I tripped up on.

File: gatsby/content/blog/2022/01/2022-02-08-synapse-1.52-released.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (babolivier) [reply gatsby/content/blog/2022/01/2022-02-08-synapse-1.52-released.mdx]: Using Dan's phrasing instead.

File: gatsby/content/blog/2022/01/2022-02-08-synapse-1.52-released.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+[Synapse 1.52](https://github.com/matrix-org/synapse/releases/tag/v1.52.0) is out! Here's what's new with this week's release.
+
+## Twisted security release
+
+The team behind [Twisted](https://twistedmatrix.com/), which is the main framework Synapse uses under the hood, recently released [Twisted 22.1](https://github.com/twisted/twisted/releases/tag/twisted-22.1.0). This version fixes a [security vulnerability](https://github.com/twisted/twisted/security/advisories/GHSA-92x2-jw7w-xvvx) within the Twisted library.
+
+While preparing the release of Synapse 1.52, we have investigated the impact of this vulnerability on Synapse. We came to the conclusion that it does not affect Synapse. We however advise server administrators to ensure they use an up-to-date version of the library as a matter of good practice.
+
+For instances installed with `pip`, the library can be updated with `pip install --upgrade Twisted`. For instances installed with the `matrixdotorg/synapse` Docker image or Debian packages from `packages.matrix.org`, updating to Synapse 1.52.0 is sufficient, as these images and packages include up-to-date versions of all dependencies.
```
Comment (babolivier) [inline gatsby/content/blog/2022/01/2022-02-08-synapse-1.52-released.mdx:17]:
```suggestion
For instances installed with `pip`, the library can be updated with `pip install --upgrade Twisted treq`. For instances installed with the `matrixdotorg/synapse` Docker image or Debian packages from `packages.matrix.org`, updating to Synapse 1.52.0 is sufficient, as these images and packages include up-to-date versions of all dependencies.
```

**Comment by babolivier** [comment]
Ignoring failing spell check since it doesn't look related to this PR.

**Pull request merged**